WtWSMS/common/landed_titles/BP_e_byzantium.txt

```diff
@@ -16455,39 +19514,7 @@ e_byzantium = {
 				}
 				b_svrljig = {
 					province = 3661
-					cultural_names = {
-						#Roman Сultures
-						name_list_roman = cn_timacum_maior
-						name_list_romano_gallic = cn_timacum_maior
-						name_list_romano_hispanic = cn_timacum_maior
-						name_list_romano_british = cn_timacum_maior
-						name_list_dalmatian = cn_timacum_maior
-						name_list_romano_african = cn_timacum_maior
-						name_list_romano_gothic = cn_timacum_maior
-						name_list_romano_dacian = cn_timacum_maior 
-						name_list_romano_frank = cn_timacum_maior 
-						name_list_romano_aramean = cn_timacum_maior 
-						name_list_romano_vandalic = cn_timacum_maior 
-						name_list_romano_thracian = cn_timacum_maior 
-						name_list_romano_illyrian = cn_timacum_maior 
-						name_list_romano_raetian = cn_timacum_maior 
-						name_list_romano_noric = cn_timacum_maior 
-						name_list_romano_pannonian = cn_timacum_maior
-						#Byzantine Cultures
-						name_list_greek = cn_timakon_megas
-						name_list_aeolian = cn_timakon_megas
-						name_list_helleno_aramean = cn_timakon_megas
-						name_list_greek_gothic = cn_timakon_megas
-						name_list_greek_persian = cn_timakon_megas
-						name_list_isaurian_persian = cn_timakon_megas
-						name_list_grecco_axumite = cn_timakon_megas
-						name_list_bosporan = cn_timakon_megas
-						name_list_pontic = cn_timakon_megas
-						name_list_tsakonian = cn_timakon_megas
-						name_list_massalian = cn_timakon_megas
-						name_list_helleno_thracian = cn_timakon_megas
-						name_list_helleno_libyan = cn_timakon_megas
-					}
```

Comment:
Why are you removing this?

Reply:
> Why are you removing this?

I used non-reliable source for this one, Timacum Maius is usually placed at Knjazevac, a city located in another CK3 county.

Reply:
Ok